website/versioned_docs/version-5.x/module-types.md

```diff
@@ -0,0 +1,51 @@
+title: single-spa conceptual module types
+sidebar_label: Conceptual Module Types
+---
+
+# Concept: single-spa modules
+
+Single-spa has [different categories](/docs/microfrontends-concept#types-of-microfrontends) of microfrontends. It is up to you where and how you use each of them. However, the single-spa core team has [recommendations](/docs/recommended-setup/#applications-versus-parcels-versus-utility-modules).
+
+Here is how each single-spa module works conceptually. This information should help you understand our [recommendations](/docs/recommended-setup/#applications-versus-parcels-versus-utility-modules).
+
+| application                       | parcel                               | utility                              |
+| --------------------------------- | ------------------------------------ | ------------------------------------ |
+| has multiple routes               | no routes                            | no route                             |
+| declarative API                   | impertive API                        | not registered/mounted by single-spa |
+| renders UI                        | renders UI                           | no UI                                |
+| single-spa managed lifecylces     | custom managed lifecycles            | no DOM elements                      |
+| Core building block               | only needed with multiple frameworks | useful to share business logic       |
+
+Each single-spa module is in reality an in browser javascript module see [here](/docs/recommended-setup#in-browser-versus-build-time-modules) for more information. Each type/module fits into the overall microfrontend approach.
+
+## Applications
+
+### Applications are declarative
+Applications use a declarative api. Your single-spa config (also sometimes called root config) defines applications ahead of time, defines the condition at which they are active, but doesn't mount the application directly.
+
+### Applications have managed lifecycles
+single-spa manages registered applications and is in charge of all of their lifecycles. This prevents you from writing a bunch of logic about when applications should mount and unmount; single-spa takes care of that for you.
+All single spa needs to make this work automatically is for an activity function that describes when your application should be active.
+
+## Parcels
+
+### Parcels are imperitive
+Parcels exist in many ways as an escape hatch from the normal declarative flow. They exist primarily to allow you to reuse UI across applications when those applications are written in multiple frameworks.
+
+### You manage the lifecycles of Parcels
+When you call `mountParcel` or `mountRootParcel` [(see api)](/docs/parcels-api) the parcel is mounted immediately and returns the parcel object. You need to call the `unmount` method on the parcel manually when the component that calls `mountParcel` unmounts.
+
+### Parcels are best suited for sharing components between frameworks
```

Comment:
"components" seems a bit too overloaded too. What do you think of

> best suited for sharing UI between frameworks.

Alternative terms I think should be considered are "widgets" or "UI components".

Reply:
I think that the term component works fine here. Parcels really are framework agnostic components.